registry/insecure.md

```diff
@@ -26,7 +26,7 @@ isolated testing or in a tightly controlled, air-gapped environment.
     `/etc/docker/daemon.json` on Linux or
     `C:\ProgramData\docker\config\daemon.json` on Windows Server. If you use
     Docker Desktop for Mac or Docker Desktop for Windows, click the Docker icon, choose
-    **Preferences**, and choose +**Daemon**.
+    **Settings**, and choose +**Docker Engine**.
```

Comment:
Thanks; perhaps then use something like;

```suggestion
    **Preferences** (Mac) or **Settings** (Windows), and choose **Docker Engine**.
```

Reply:
I think it's a good idea too.